From 7e50754a7c91415bbd8ea5e0519fb815b6418305 Mon Sep 17 00:00:00 2001
From: Beth Thibodeau <ethibodeau@google.com>
Date: Sat, 28 Oct 2023 00:46:24 +0000
Subject: [PATCH] Add null check for media metadata

This check was inadvertently removed in a previous cherry pick and is still necessary.

Bug: 307983823
Test: build
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:abc33695709d44d3f23a310601070d496067950a)
Merged-In: I35e89feaf89322babb89fe0a30430606ebdfcdfc
Change-Id: I35e89feaf89322babb89fe0a30430606ebdfcdfc
---
 .../SystemUI/src/com/android/systemui/media/MediaDataManager.kt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/media/MediaDataManager.kt b/packages/SystemUI/src/com/android/systemui/media/MediaDataManager.kt
index 70af5b647362..fba7ac037ee8 100644
--- a/packages/SystemUI/src/com/android/systemui/media/MediaDataManager.kt
+++ b/packages/SystemUI/src/com/android/systemui/media/MediaDataManager.kt
@@ -574,7 +574,7 @@ class MediaDataManager(
         // Song name
         var song: CharSequence? = metadata?.getString(MediaMetadata.METADATA_KEY_DISPLAY_TITLE)
         if (song.isNullOrBlank()) {
-            song = metadata.getString(MediaMetadata.METADATA_KEY_TITLE)
+            song = metadata?.getString(MediaMetadata.METADATA_KEY_TITLE)
         }
         if (song.isNullOrBlank()) {
             song = HybridGroupManager.resolveTitle(notif)
-- 
2.43.0.rc1.413.gea7ed67945-goog

